### PR TITLE
Custom view

### PR DESCRIPTION
**Pull request by HebaMad**

### CODE REVIEW

File: DokanStore/Networking/Package.swift [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+         .iOS(.v10),
+     ],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "Networking",
+            targets: ["Networking"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/Moya/Moya.git", .upToNextMajor(from: "15.0.0"))
```
Comment (ahmdyasser) [inline DokanStore/Networking/Package.swift:19]: We dropped Moya and now using Alamofire instead

File: DokanStore/DokanStore/XIB/CustomView.swift [hunk 2]
```diff
@@ -0,0 +1,25 @@
+//
+//  CustomView.swift
+//  DokanStore
+//
+//  Created by heba isaa on 18/06/2022.
+//
+
+import UIKit
+
+class CustomView: UIViewFromNib {
```
Comment (ahmdyasser) [inline DokanStore/DokanStore/XIB/CustomView.swift:10]: The class should have a more specific name

File: DokanStore/DokanStore/XIB/CustomView.swift [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    @IBOutlet weak var valueTF: UITextField!
+    @IBOutlet weak var titleTxt: UILabel!
+    
+    @IBInspectable var text:String?{
```
Comment (ahmdyasser) [inline DokanStore/DokanStore/XIB/CustomView.swift:15]:
```suggestion
    @IBInspectable var text: String? {
```
also check Swift guidelines for spacing
https://github.com/raywenderlich/swift-style-guide

**Review by ahmdyasser — COMMENTED** [review]
@ahmdmhasn I also don't know why the .resolved file shows here? It already has been to the .gitignore file.

File: DokanStore/Networking/Package.swift (hunk 1 above)
Comment (HebaMad) [reply DokanStore/Networking/Package.swift:19]: I told Eng.Ahmed I want to use Moya

File: DokanStore/DokanStore/XIB/CustomView.swift (hunk 2 above)
Comment (HebaMad) [reply DokanStore/DokanStore/XIB/CustomView.swift:10]: this is a general Method for a UIView

File: DokanStore/DokanStore/XIB/CustomView.swift (hunk 3 above)
Comment (HebaMad) [reply DokanStore/DokanStore/XIB/CustomView.swift:15]: 👍🏻👍🏻

**Review by ahmdmhasn — CHANGES_REQUESTED** [review]
We still need to discuss the addition of files that aren't supposed to be added 😁

File: DokanStore/Networking/Package.swift (hunk 1 above)
Comment (ahmdyasser) [reply DokanStore/Networking/Package.swift:19]: Yeah, sorry I thought this would be merged to the main branch

**Pull request closed without merging**